frontend/amplify/package.json

```diff
@@ -4,6 +4,14 @@
   "private": true,
   "main": "index.ts",
   "type": "module",
+  "scripts": {
+    "generate-dependencies": "tsx functions/send-email/generate-dependencies.ts",
+    "lint": "eslint . -c .eslintrc.json",
+    "lint:fix": "eslint . --fix",
+    "test:unit": "jest",
+    "typecheck": "tsc --noEmit",
+    "typescript": "^5.5.4"
```

Comment:
How comes this is in scripts and not dependencies?

Reply:
🤔

Reply:
Good spot. It was already in `scripts` and I just moved the entire `scripts` block. But I will fix it.

Reply:
fixed